src/builtin/tables.scrbl

```diff
@@ -47,53 +67,150 @@ my-table = table: name :: String, age :: Number, favorite-color :: String
 end
   }
 
-Note that @pyret{my-table} is referred to in many of the following examples.
+Entering the name of a table in the interactions window after running the program
```

Comment:
Not quite.  Tables don't have names, any more than other values do.  (It's just as weird as saying that because you write `x = 5`, that 5 has the name "x".)